x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -169,6 +177,45 @@ export function TimeseriesChart({
             />
           );
         })}
+
+        {anomalySeries &&
+          anomalySeries.map((anomalySerie) => {
+            if (anomalySerie.type === 'area') {
+              return (
+                <AreaSeries
+                  key={anomalySerie.title}
+                  id={anomalySerie.title}
```

Comment:
It's also a "series" even if it's just one (there are still multiple datapoints). Not about a better naming to distinguish between the list of series vs a single series though.